kolibri/core/assets/src/navigation.vue

```diff
@@ -1,33 +1,12 @@
 <template>
-
   <header role="banner">
     <nav class="titlebar" role="navigation" aria-label="Main navigation">
```

Comment:
That would be preferable yes, for the sake of more fluid user experience, but more than one in itself does not represent an error.